financial-connections/src/main/java/com/stripe/android/financialconnections/FinancialConnectionsSheetViewModel.kt

```diff
@@ -119,11 +121,23 @@ internal class FinancialConnectionsSheetViewModel @Inject constructor(
     private fun initAuthFlow() {
         viewModelScope.launch {
             kotlin.runCatching {
-                val attestationInitialized = prepareStandardRequestManager()
-                getOrFetchSync(
+                val attestationInitResult = prepareStandardRequestManager()
+                val syncResponse = getOrFetchSync(
                     refetchCondition = Always,
-                    attestationInitialized = attestationInitialized
+                    attestationInitialized = attestationInitResult.initialized
                 )
+                val pane = syncResponse.manifest.nextPane
+                analyticsTracker.track(
```

Comment:
One potential worry is that we'll emit an event on all sessions, after the first sync call.